plugins/module_utils/constants.py

```diff
@@ -211,3 +211,31 @@
 
 ORIGINATE_DEFAULT_ROUTE = {"only": "only", "in_addition": "inAddition", "": ""}
 L3OUT_ROUTING_PROTOCOLS = {"bgp": ["bgp"], "ospf": ["ospf"], "bgpOspf": ["bgp", "ospf"], None: [None], "": None, "bgpospf": "bgpOspf", "ospfbgp": "bgpOspf"}
+
+TARGET_COS_MAP = {
+    "background": "cos0",
+    "best_effort": "cos1",
+    "excellent_effort": "cos2",
+    "critical_applications": "cos3",
+    "video": "cos4",
+    "voice": "cos5",
+    "internetwork_control": "cos6",
+    "network_control": "cos7",
+    "unspecified": "cos8",
+}
+
+DSCP_KEYS_FORMAT_MAP = {
+    "dscp_from": ["dscpFrom", TARGET_DSCP_MAP],
```

Comment:
did you consider making this a named tuple or data class to increase readability in the function?

this way in line `formatted_dict[value[0]] = d[key] if d.get(key) in value[1].values() else value[1].get(d.get(key))`, the value[0] and value[1] would be replaced with something that is easier to understand